### PR TITLE
Change podsecuritypolicy description from PREVIEW to DEPRECATING.

### PR DESCRIPTION
**Pull request by JunSun17**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [ ] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json [hunk 1]
```diff
@@ -1746,7 +1746,7 @@
         },
         "enablePodSecurityPolicy": {
           "type": "boolean",
-          "description": "(PREVIEW) Whether to enable Kubernetes Pod security policy."
+          "description": "(DEPRECATING) Whether to enable Kubernetes Pod security policy."
```
Comment (jluk) [inline specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json:1749]: @JunSun17 could we actually update this to:

(DEPRECATING) Whether to enable Kubernetes pod security policy (preview). This feature is set for removal on October 15th, 2020. Learn more at aka.ms/aks/azpodpolicy.

**Review by jluk — CHANGES_REQUESTED** [review]
small ask to clarify the description with more details

@palma21 / @robbiezhang FYI

**Comment by jluk** [comment]
@nullmdr please take a look at this, AKS is deprecating a preview feature and we would like to signal a heads up via Swagger - we have already made public disclosure and direct email notifications for 90 days notice. Exposing this in details for swagger is an additional step to help customers get the message.

File: specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json (hunk 1 above)
Comment (JunSun17) [reply specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-02-01/managedClusters.json]: done.

**Review by gtxistxgao — APPROVED** [review]

**Review by jluk — APPROVED** [review]

**Comment by yungezz** [comment]
hi @junsun17 pls fix CI failure with instruction https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/ci-fix.md, then I'll merge the PR.

**Pull request merged**